projects/documentation/content/migrations/2024-10-01 (1.0.0).md

```diff
@@ -0,0 +1,189 @@
+---
+layout: guide.njk
+title: 'Migration Guide: Spectrum Web Components (v1.0.0)'
+displayName: 2024/10/01 (v1.0.0)
+slug: migration-guide
+---
+
+## Migration: 2024/10/01 (v1.0.0)
+
+With the release of Spectrum Web Components version 1.0.0, we removed previously deprecated components and features.
+Starting with this version, Spectrum Web Components adheres to [semantic versioning (semver)](https://semver.org/), ensuring that any future breaking changes will be introduced only in major version updates. These changes will be communicated in advance, allowing developers to prepare and avoid disruption to existing applications. Patch releases, as part of this versioning approach, will be non-breaking and focus on fixes and minor improvements.
```

Comment:
```suggestion
With the release of Spectrum Web Components v1.0.0, we've removed previously deprecated components and features.

Starting with this version, Spectrum Web Components will adhere to [semantic versioning (semver)](https://semver.org/), which ensures that any future breaking changes are introduced only in major version bumps. These changes will be communicated in advance so that developers can prepare for the updates. As part of this versioning approach, patch releases will be non-breaking and will focus on fixes and minor improvements.
```

Minor tweaks